shared/templates/mount_option_home/oval.template

```diff
@@ -0,0 +1,77 @@
+{{% set non_allowed_partitions = ["/lib",
```

Comment:
In a situation where the interactive user home directory is a sub-folder of a "not allowed" mount point, the check wouldn't pass even if the partition have the mount `option` set. Maybe the template should not consider the "not allowed" partitions for the check. The main risk is about the remediation, which shouldn't change critical partitions.

So, I believe the OVAL check could be simplified by removing the `test_home_allowed_partition_{{{ MOUNTOPTION }}}` and keeping this control only in the remediation level. I am open to more opinions.

Reply:
agree